tools/cryptogenotyper/cryptogenotyper.xml

```diff
@@ -0,0 +1,141 @@
+<tool id="CryptoGenotyper" name = "CryptoGenotyper" version="@VERSION@">
+  <description>
+    CryptoGenotyper is a standalone tool to analyze Cryptosporidium Sanger sequencing data and classify the species and subtype based on SSU rRNA and gp60 gene markers, respectively.
+  </description>
+  <macros>
+   <token name="@VERSION@">1.0</token>
+  </macros>
+  <requirements>
+    <requirement type="package" version ="1.0">cryptogenotyper</requirement>
```

Comment:
Could you replace this with `@VERSION@` too?

Reply:
Sure, I forgot about this tag too. Thank you